Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -13,9 +13,10 @@ public sealed class MigrationRunner
         /// <summary>
         /// The list of known migrations, in order of applicability.
         /// </summary>
-        internal static readonly IUpdater[] Migrations =
+        internal static readonly IMigrationRoutine[] Migrations =
         {
-            new Routines.DisableTranscodingThrottling()
+            new Routines.DisableTranscodingThrottling(),
+            new Routines.CreateUserLoggingConfigFile()
```

Comment:
I still think we should move to reflection at some point, especially since we used names to possibly accomodate plugin migrations in the future.

Reply:
Yeah, I would prefer reflection as well I think. We can migrate to that at a later date without any impact though, so it might as well be a separate PR